internal/v1/api.yaml

```diff
@@ -106,6 +106,17 @@ paths:
             default: 0
             minimum: 0
           description: composes page offset, default 0
+        - in: query
+          name: ignoreImageTypes
+          # ['rhel-edge-installer', 'rhel-edge-commit', ...]
+          required: false
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/ImageTypes'
+          description: filter image_type for getComposes end point.|
+                                   This filter is general, and not mandatory so user can choose
+                                   which image type he wants to filter.
```

Comment:
How about
```
description: |
  Filter  the composes on image type. The filter is optional and can be specified multiple times.
```

That way we avoid gendered language too :)

Reply:
done